dredd/dredd_hook.py

```diff
@@ -89,7 +93,7 @@ def configure_transaction(transaction):
         new_url = path
         for name, value in params.items():
             value = evaluate(value)
-            new_url = new_url.replace('{' + name + '}', str(value))
+            new_url = new_url.replace('{' + name + '}', binary_type(value))
```

Comment:
@sharkykh replace `binary_type(value)` with `str(value)`. This is the reason because dreed is taking 13 minutes to fail. Please, enable dreed for py3.

Reply:
Done, however the reason it takes 13 minutes to fail is because Medusa cannot be loaded using Python 3.6 yet, so the API tests try to run but the backend server (Medusa) isn't available, so each test times out, which takes time.